Resources/Prototypes/Procedural/salvage_loot.yml

```diff
@@ -5,6 +5,16 @@
   loots:
     - !type:RandomSpawnsLoot
       entries:
+        - proto: GunSafeShotgunKammerer
+          prob: 0.75
+        - proto: CrateArmoryPistols
+          prob: 0.75
+        - proto: CrateArmoryShotgun
+          prob: 0.3
+        - proto: CrateArmorySmg
+          prob: 0.3
+        - proto: VendingMachineArcadia
+          prob: 0.5
```

Comment:
the entries all need a cost associated to them like the others since it appears they are being taken from a pool of points. And since these are so drastically more powerful than the rest of the loot, they should probably have a higher cost to each of them so that you dont get 5-6 crates every shift